guides/moving-to-oss.md

```diff
@@ -0,0 +1,235 @@
+| **VM workspaces**                                                    | [EC2 containers only](https://coder.com/docs/coder/latest/admin/workspace-providers/deployment/ec2#prerequisites) | ✅ Any spec via the Terraform [template](https://coder.com/docs/coder-oss/latest/templates)                          |
+| **Linux workspaces**                                                 | ✅                                                                                                                | ✅                                                                                                                   |
+| **Windows workspaces**                                               | ✅                                                                                                                | ✅                                                                                                                   |
+| **macOS workspaces**                                                 | ❌                                                                                                                | ✅                                                                                                                   |
+| **ARM workspaces**                                                   | ❌                                                                                                                | ✅                                                                                                                   |
+| **Additional resources in workspace (volume mounts, API keys, etc)** | ❌                                                                                                                | ✅ Any [Terraform resource](https:///registry.terraform.io)                                                          |
+| **Workspace options**                                                | ✅ Hardcoded options                                                                                              | ✅ Any options via [template parameters](https://coder.com/docs/coder-oss/latest/templates#parameters)               |
+| **Edit workspace**                                                   | ✅                                                                                                                | ⌛ [#802](https://github.com/coder/coder/issues/802)                                                                 |
+| **Resource provisoning rates**                                       | ✅ Organization wide                                                                                              | ✅ Template wide [(needs docs)](https://github.com/coder/coder/issues/3519)                                          |
```

Comment:
```suggestion
| **Resource over-provisoning ratios**                                       | ✅ Organization wide                                                                                              | ✅ Template wide [(needs docs)](https://github.com/coder/coder/issues/3519)                                          |
```

Reply:
We've opted to call them provisioning rates 

<img width="315" alt="Screen Shot 2022-09-09 at 12 51 09 PM" src="https://user-images.githubusercontent.com/22407953/189413596-70397f0f-f5bb-4dbf-ba0c-746ce6d157b0.png">

Reply:
I mean I guess, but it's not a good name.